modules/tinymce/src/core/main/ts/api/EditorCommands.ts

```diff
@@ -52,81 +39,49 @@ export interface EditorCommandsConstructor {
 
 class EditorCommands {
   private readonly editor: Editor;
-  private selectionBookmark: Bookmark;
   private commands: Commands = { state: {}, exec: {}, value: {}};
 
   public constructor(editor: Editor) {
     this.editor = editor;
-
-    this.setupCommands(editor);
   }
 
   /**
-   * Executes the specified command.
+   * Executes a registered command on the current instance. A list of available commands can be found in
+   * the <a href="https://www.tiny.cloud/docs/advanced/editor-command-identifiers/">command identifiers</a> documentation.
```

Comment:
This docs link is almost certainly going to break for 6.0, as the docs structure is changing. Can we please make sure we have a followup Jira to fix this or change it to something else that's less hardcoded? At the very least we should remove the domain, as otherwise it won't work on staging for example.

- [ ] Replace with something else or at the very least remove the domain in the docs URLs

Reply:
I removed the site link we could improve this later when the new docs site is up. I guess we might need to go though it all I think we have links else were like the UI registry API docs for example.

Reply:
Looks like Mike has logged TINY-8441 to do this.